scripts/docker-integration-tests/cold_write_no_index/test.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source $GOPATH/src/github.com/m3db/m3/scripts/docker-integration-tests/common.sh
+REVISION=$(git rev-parse HEAD)
+
+echo "Run m3dbnode docker container"
+
+CONTAINER_NAME="m3dbnode-version-${REVISION}"
+docker create --name "${CONTAINER_NAME}" -p 9000:9000 -p 9001:9001 -p 9002:9002 -p 9003:9003 -p 9004:9004 -p 7201:7201 "m3dbnode_integration:${REVISION}"
+
+# think of this as a defer func() in golang
+function defer {
+  echo "Test complete, dumping logs"
+  echo "---------------------------"
+  docker logs "${CONTAINER_NAME}"
+  echo "---------------------------"
+
+  echo "Removing docker container"
+  docker rm --force "${CONTAINER_NAME}"
+}
+trap defer EXIT
+
+docker start "${CONTAINER_NAME}"
+
+# TODO(rartoul): Rewrite this test to use a docker-compose file like the others so that we can share all the
+# DB initialization logic with the setup_single_m3db_node command in common.sh like the other files. Right now
+# we can't do that because this test doesn't use the docker-compose networking so we have to specify 127.0.0.1
+# as the endpoint in the placement instead of being able to use dbnode01.
+echo "Wait for DB to be up"
+ATTEMPTS=10 MAX_TIMEOUT=4 TIMEOUT=1 retry_with_backoff  \
+  'curl -vvvsSf 0.0.0.0:9002/bootstrappedinplacementornoplacement'
+
+echo "Wait for coordinator API to be up"
+ATTEMPTS=10 MAX_TIMEOUT=4 TIMEOUT=1 retry_with_backoff  \
+  'curl -vvvsSf 0.0.0.0:7201/health'
+
+echo "Adding namespace"
+curl -vvvsSf -X POST 0.0.0.0:7201/api/v1/namespace -d '{
+  "name": "unagg",
+  "options": {
+    "bootstrapEnabled": true,
+    "flushEnabled": true,
+    "writesToCommitLog": true,
+    "cleanupEnabled": true,
+    "snapshotEnabled": true,
+    "repairEnabled": false,
+    "coldWritesEnabled": true,
+    "retentionOptions": {
+      "retentionPeriodDuration": "8h",
+      "blockSizeDuration": "2h",
+      "bufferFutureDuration": "10m",
+      "bufferPastDuration": "10m",
+      "blockDataExpiry": true,
+      "blockDataExpiryAfterNotAccessPeriodDuration": "5m"
+    }
+  }
+}'
+
+echo "Sleep until namespace is init'd"
+ATTEMPTS=4 TIMEOUT=1 retry_with_backoff  \
+  '[ "$(curl -sSf 0.0.0.0:7201/api/v1/namespace | jq .registry.namespaces.unagg.indexOptions.enabled)" == false ]'
+
+echo "Placement initialization"
+curl -vvvsSf -X POST 0.0.0.0:7201/api/v1/placement/init -d '{
+    "num_shards": 4,
+    "replication_factor": 1,
+    "instances": [
+        {
+            "id": "m3db_local",
+            "isolation_group": "rack-a",
+            "zone": "embedded",
+            "weight": 1024,
+            "endpoint": "127.0.0.1:9000",
+            "hostname": "127.0.0.1",
+            "port": 9000
+        }
+    ]
+}'
```

Comment:
Can you use the `/database/create` API instead of namespace and placement btw?  

Trying to migrate most docs and tests to use that instead (so more users use it and have less issues with block size, etc).

Is there a way to create database without index turned on with the `/database/create` endpoint?  I guess that would be the only potential issue.

Reply:
Unfortunately it's not one of the options for the `/create` endpoint, which should probably stay that way, since that endpoint should contain only the recommended options in there.

Reply:
Yeah, I'm inclined to agree.